taxonomies/additives.txt

```diff
@@ -9554,28 +9924,42 @@ wikidata:en: Q4747303
 # comment:en:E350 is written in plural to distinguish it from E350i.
 en: E350, Sodium malates
 xx: E350
-bg: E350, E350 food additive
+ar: E350, مالات الصوديوم
+az: E350, Natrium malatlar
+bg: E350, Натриеви малати
 ca: E350, Malat de sodi
-cs: E350, E350 food additive
-da: E350, E350 food additive
+cs: E350, Jablečnan sodný
+da: E350, Natriummalater
 de: E350, Natriummalate
 el: E350, άλατα του μηλικού οξέος με νάτριο
 es: E350, malatos de sodio
-et: E350, E350 food additive
+et: E350, Naatriummalaadid
 fi: E350, natriummalaatit
 fr: E350, malates de sodium, malates
+he: E350, מלאטים של נתרן
+hi: E350, सोडियम मैलेट
+hr: E350, Natrijevi malati
```

Comment:
2025-03-07T21:50:07.0345490Z ERROR - duplicate_synonym - 11738 - hr:natrijevi-malati already is a synonym of hr:e350 for entry en:e350 (ingredients) - hr:natrijevi-malati cannot be mapped to entry en:e350i / hr:e350i